src/actions/categories/createCategoryAsyncThunk.ts

```diff
@@ -0,0 +1,13 @@
+import { createAsyncThunk } from '@reduxjs/toolkit';
+import axios from 'axios';
+import { CategoryModel } from '../../models/CategoryModel';
+
+
+
+export const createCategoryAsyncThunk = createAsyncThunk(
+    'catalog/createCategoryAsyncThunk',
+     async (category: CategoryModel) => {
+         const response = await axios.post(`${process.env.REACT_APP_API_URL}/api/categories`, category)
```

Comment:
It's a bad practice to send the requests in the think action directly. Move this code to the ApiServices. For example 
![image](https://user-images.githubusercontent.com/116076244/207028871-99c086da-71eb-459b-9310-2a170ea1dd57.png)
![image](https://user-images.githubusercontent.com/116076244/207029067-78a60934-48ba-4336-af59-e99b7679f5a3.png)